utils/script-native.c

```diff
@@ -0,0 +1,106 @@
+#ifdef HAVE_LIBNATIVE
+
+#define PR_FMT    "script"
+#define PR_DOMAIN DBG_SCRIPT
+
+#include <dlfcn.h>
+#include "utils/utils.h"
+#include "utils/symbol.h"
+#include "utils/fstack.h"
+#include "utils/filter.h"
+#include "utils/script.h"
+#include "utils/script-native.h"
+
+static void *native_handle;
+
+static void (*__uftrace_begin)(struct script_info *info);
+static void (*__uftrace_entry)(struct script_context *sc_ctx);
+static void (*__uftrace_exit)(struct script_context *sc_ctx);
+static void (*__uftrace_end)(void);
+
+static int native_uftrace_begin(struct script_info *info)
+{
+	__uftrace_begin(info);
+	return 0;
+}
+
+static int native_uftrace_entry(struct script_context *sc_ctx)
+{
+	__uftrace_entry(sc_ctx);
+	return 0;
+}
+
+static int native_uftrace_exit(struct script_context *sc_ctx)
+{
+	__uftrace_exit(sc_ctx);
+	return 0;
+}
+
+static int native_uftrace_end(void)
+{
+	__uftrace_end();
+	return 0;
+}
+
+static int native_atfork_prepare(void)
+{
+	fflush(stdout);
```

Comment:
It's already done in the `atfork_prepare_handler()`.

Reply:
Right. I removed it.